src/pages/settings/PasswordChangePage.tsx

```
@@ -1,26 +1,116 @@
-    // TODO(API 연결 시): PATCH /users/password
-    try {
-      console.log("submit password change");
+  const newPassword = useWatch({ control, name: "newPassword" });
+  const newPasswordCheck = useWatch({ control, name: "newPasswordCheck" });
+  const currentPassword = useWatch({ control, name: "currentPassword" });
+  const [serverCurrentPwError, setServerCurrentPwError] = useState("");
+  const [serverNewPwCheckError, setServerNewPwCheckError] = useState("");
 
-      setSuccessOpen(true);
-    } finally {
+  useEffect(() => {
```

Biome (2.3.13)

[error] This hook specifies more dependencies than necessary: newPassword.

This dependency can be removed from the list.

React relies on hook dependencies to determine when to re-compute Effects.
Specifying more dependencies than required can lead to unnecessary re-rendering
and degraded performance.
Unsafe fix: Remove the extra dependencies from the list.


(lint/correctness/useExhaustiveDependencies)